cli/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "xstate-cli",
+  "version": "0.0.1",
+  "bin": {
+    "xstate": "./bin/index.js"
+  },
+  "scripts": {
+    "prepare": "esbuild src/bin.ts --outfile=bin/index.js --bundle --platform=node --target=node14 --external:fsevents"
+  },
+  "devDependencies": {
+    "@types/node": "^16.4.0",
+    "esbuild": "^0.12.22",
+    "xstate": "^4.29.0"
+  },
+  "peerDependencies": {
+    "xstate": "^4.29.0"
+  },
+  "dependencies": {
+    "chokidar": "^3.5.3",
+    "commander": "^8.0.0",
+    "fast-glob": "^3.2.11",
+    "xstate-parser-demo": "0.5.3",
```

Comment:
what's that? is this pkg really a dependency of the `xstate-cli`?

Reply:
Yes, we need to parse the code into machines. xstate-parser currently on npm as `xstate-parser-demo`, so that's why it's imported.

Reply:
ah, OK - the package name threw me off

Reply:
We can move this to `@xstate/parser`

Reply:
it should be something like `@xstate/machines-extractor`, `@xstate/miner`, or something like this - as calling it a parser is a tiny stretch